<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkDialog</class>
 <widget class="QDialog" name="WorkDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Works</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../halturka.qrc">
    <normaloff>:/images/work.png</normaloff>:/images/work.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>470</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="firstButton">
      <property name="text">
       <string>&lt;&lt; &amp;First</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="previousButton">
      <property name="text">
       <string>&lt; &amp;Previous</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>&amp;Next &gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="lastButton">
      <property name="text">
       <string>&amp;Last &gt;&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="statusCheckBox">
      <property name="text">
       <string>Completed</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="dateStartLabel">
      <property name="text">
       <string>Date start:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="dateStartEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="dateFinishLabel">
      <property name="text">
       <string>Date finish:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="dateFinishEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="priceLabel">
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="priceEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="customerLabel">
      <property name="text">
       <string>Customer:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="customerComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>269</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>&amp;Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>&amp;Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>&amp;Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>formLayoutWidget_2</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>customerComboBox</zorder>
  <zorder>customerLabel</zorder>
  <zorder>priceLabel</zorder>
  <zorder>priceEdit</zorder>
 </widget>
 <resources>
  <include location="../halturka.qrc"/>
 </resources>
 <connections/>
</ui>
